packages/sql-mapper/test/cursor.test.js

```diff
@@ -0,0 +1,570 @@
+'use strict'
+
+const { test } = require('node:test')
+const { equal, deepEqual, ifError } = require('node:assert')
+const { connect } = require('..')
+const { clear, connInfo, isSQLite } = require('./helper')
+const errors = require('../lib/errors')
+
+const fakeLogger = {
+  trace: () => {},
+  error: () => {},
+  warn: () => {},
+}
+
+test('single field cursor pagination', async () => {
+  const mapper = await connect({
+    ...connInfo,
+    log: fakeLogger,
+    async onDatabaseLoad (db, sql) {
+      test.after(async () => {
```

Comment:
You should call after on the test context passed to the test callback itself.

Reply:
+